.github/workflows/ibis-main.yml

```diff
@@ -92,11 +94,11 @@ jobs:
 
       - name: run all core tests and run benchmarks once parallel
         if: matrix.os != 'windows-latest'
-        run: just ci-check -m "'core or benchmark'" -n auto
+        run: poetry run pytest -v -n auto ibis/common ibis/expr ibis/tests ibis/formats
```

Comment:
Can you add a TODO to bring the `just ci-check` version?

Reply:
Added